roles/ethereum_node/defaults/main.yaml

```diff
@@ -4,7 +4,7 @@ ethereum_node_images_always_pull: false
 ethereum_node_announced_ip: "{{ ansible_host }}"
 
 # Execution layer client
-ethereum_node_el: ethereumjs # Valid options: besu, geth, erigon, nethermind
+ethereum_node_el: reth # Valid options: besu, geth, erigon, nethermind, ethereumjs, reth
```

Comment:
uhh why is the default being reth? can we leave it as geth or something that we recommend for prod setups?

Reply:
yeah gonna change that back to geth.